### ISSUE TITLE
Button between send and receive is unlabeled

### ISSUE THREAD

**Issue by tspivey** [mycelium-com/wallet#296]
On the balance tab of the wallet, the button between Send and Receive is unlabeled.
It looks like this is an ImageView, which is lacking a content description. Without this, when I navigate to it with TalkBack (the built-in screen reader), I hear "button, unlabeled".

Once I activate it, I get put on another screen with three more unlabeled buttons.

**Comment by DanielWeigl** [comment]
Thank you @tspivey for the feedback, we will add labels for the next update.

Its the "quick QR Scan"-button, that allows you to scan a QR-code and the action is depending on what type of QR code got scanned (eg. Address or xPub->Send to, Private Key->Spend from, etc...)
The button is a ImageView with a camera as icon to symbolize QR-code scanning.

The activity that gets launched is the QR-Code scanner, the two unlabeled buttons are "Continuous Focus/Single Shot focus" and "Switch on camera flash/led"

Just out of curiosity, how does QR-Code scanning work for you (if at all)? Is there something we could improve?

**Comment by tspivey** [comment]
QR scanning doesn't work very well for me at all.
If these buttons are toggles, then I have no idea of their status.
Here is the process I followed.
1. I went to http://bitcoinqrcode.org/ and generated a QR code with one 
of my addresses. Managed to get it uploaded to Imgur.
2. Went to the resulting link, which should have just been the QR code.
3. Maximized the browser window.

Then on the phone:
1. Pressed send coins. This is the most common situation in which I 
would like to scan a QR code.
2. Pressed the Scan QR code button.

 From here, my screen reader said my phone's orientation switched to 
landscape.

Now I had the screen with the three unlabeled buttons.
I pointed the phone at the screen, and started moving it.
Nothing.
I don't know if I should have rotated the screen to match the 
orientation, I assume the scanner corrects for rotation errors.

Then I pressed one of the buttons. If it toggled something, I had no 
indication of it.

I couldn't get the code to scan. Unfortunately, it could be any number 
of things, and I'm not yet sure what.

**Comment by Giszmo** [comment]
I'm afraid, scanning a QR-code is as much a visual thing as it can get. Depending on your device, it might already be hard to scan even with visual feedback. My laptop's cam is upside down and low res, so even if positioned such that the code fills the camera view, it takes a second to scan it. With the upside-down cam I have huge difficulties to position the code right.

Many users also struggle even with good phones and a common issue is to get too close to the image. And until the code is detected, there is no feedback we could give, neither.

The buttons should normally not be needed but should also get labels in the scope of this issue. They are:

# switch cam. Normally the primary, backside cam is the default, with one exception where the frontside cam it is. On tablets with just one cam, this button should have no effect (if shown at all)
# the focus button so far never was necessary for me
# the flashlight button should not be needed for scanning from the screen, neither. Not sure if there is any circumstance under which this might be on by default, which might degrade the screen scanning performance.

**Comment by DanielWeigl** [comment]
Maybe its easier for you to not use QR at all. It seems you have an address on a desktop computer - or? Can you mail/skype/telegramm it to your phone? If so, you can copy it and then in mycelium on the main-screen you can choose the button "Send" and then "Clipboard"  (that button is usually disabled, if the clipboard content is not an address, maybe your screen reader does not mention the button then)

**Comment by Giszmo** [comment]
I'd also wonder if addresses can always be read for verification easily and how the backup and restore screens with their custom keyboard work.

**Comment by tspivey** [comment]
The clipboard works well.
The screen reader tells me that the Clipboard button is disabled, until 
I copy an address.

For reading addresses for verification, I can set the screen reader to 
read by character and read through them that way

I was able to backup my wallet, and the keyboard seemed to work fine for 
verification.

**Comment by Giszmo** [comment]
Trying out the app with screen reader (TalkBack) I found several issues but a user who knows apps that do much better for blind people could probably best give insight on what really is worth fixing. Also I guess there is more low hanging fruits for making Mycelium really accessible by a non-bitcoin non-Mycelium developer with experience in this kind of accessibility
